airbnb/src/components/Calendar/CalendarBody.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import CalendarDates from './CalendarDates';
+
+function CalendarBody(props) {
```

Comment:
```suggestion
function CalendarBody({totalDate}) {
```
바로 destructuring해서 꺼내주셔도 괜찮을 것 같네요